rum/rum-android-mobile.mdx

```diff
@@ -0,0 +1,294 @@
+      | :----------------------:       | :-------------:| :-----------------------------------------------------------------------------  |
+      | `component`                    |      String    |  Sets `ui`                                                                      |
+      | `activityName`, `activity.name`|      String    |  Name of the activity class. Example: `MainActivity`                            |
+    </Accordion>
+
+    <Accordion title="Fragment Lifestyle Monitoring">
+      Fragment lifecycle monitoring involves generating spans whenever an fragment undergoes a state change. The name of a fragment lifecycle span can vary based on its state::
+
+      - `Created`: Activity starts for the first time.
+
+      - `Restarted`:Activity restarts after being stopped.
+
+      - `Resumed`: Activity resumes after a pause.
+
+      - `Paused`: Activity is paused.
+
+      - `Stopped`: Activity stops.
+
+      - `Destroyed`: Activity is destroyed.
+
+      |         __Name__               |    __Type__    |                     __Description__                                             | 
+      | :----------------------:       | :-------------:| :-----------------------------------------------------------------------------  |
+      | `component`                    |      String    |  Sets `ui`                                                                      |
+      | `fragmentName`                 |      String    |  Name of the fragment class Example: `MainFragment`                             |
+    </Accordion>
+
+    <Accordion title="App Start Monitoring">
+      The app start monitoring feature creates spans each time the app undergoes a cold, warm, or hot start.
+
+      - `Cold Starts`: Occurs when users open the app for the first time after booting their phone or terminating the app.
+
+      - `Warm Starts`: Occurs when some operations from a cold start are still in progress. Warm starts are quicker than cold starts but slower than hot starts.
+
+      - `Hot Starts`: Occurs when the system brings an app to the foreground. They are faster than cold starts as the app is already loaded.
+
+      App start monitoring generates spans named AppStart with associated attributes:
+
+      |         __Name__               |    __Type__    |                     __Description__                                             | 
+      | :----------------------:       | :-------------:| :-----------------------------------------------------------------------------  |
+      | `component`                    |      String    |  Sets `appstart`                                                                |
+      | `start.type`                   |      String    |  Type of start. Value `cold`, `warm`, or `hot`                                  |
+    </Accordion>
+
+  </Accordion>
+
+
+</AccordionGroup>
+
+<Note> Need assistance or want to learn more about Middleware? Contact us at support[at]middleware.io. </Note>
```

Comment:
This doesnt actually render the `@` symbol for me. idk if thats just mintlify tho.